diff --git a/node_modules/hexo-theme-icarus/include/style/base.styl b/node_modules/hexo-theme-icarus/include/style/base.styl
index 6c3bbb9..f723bf9 100644
--- a/node_modules/hexo-theme-icarus/include/style/base.styl
+++ b/node_modules/hexo-theme-icarus/include/style/base.styl
@@ -46,7 +46,7 @@ $menu-item-active-background-color ?= hsl(219, 70%, 96%)
 $content-heading-weight ?= $weight-normal
 
 
-$logo-height ?= 1.75rem
+$logo-height ?= 5rem
 
 // FIXME: https://github.com/groenroos/bulma-stylus/issues/11
 @import bulma-stylus-root + '/utilities/initial-variables'
@@ -97,3 +97,81 @@ body > .section
 
     ::-webkit-scrollbar-thumb:hover
         background: rgba(0,0,0,0.24)
+
+.my-hightlight 
+        /* 前 50% 是透明 ( transparent ) ，後 50% 才有顏色 */
+        background-image: linear-gradient(transparent 50%, #5ab1ea78 50%);
+        /* 做偏移，依需求而定 */
+        margin: 0rem 0.25rem;
+        padding: 0rem 0.25rem;
+
+.content
+    h2
+        font-weight: 700;
+        color: #3398dc;
+        border-left: 5px solid;
+        padding-left: 5px;
+
+.content
+    h3,h4
+        font-weight: 600;
+        color: #3398dc;
+    ul
+        margin-top 1em;
+        margin-bottom: 1em;
+
+#網路參考文章
+     background-color: #ffdd57;
+     color:black;
+     border:0px;
+     border-radius:5px 5px 0px 0px;
+     margin-bottom:0px;
+     font-size: 1.2em;
+     font-weight: 500;
+     padding:8px;
+
+.ref
+    border:3px solid #ffdd57;
+    padding:5px;
+    background-color: #fffbeb;
+    border-radius:0px 0px 5px 5px;
+    &::before
+        content:"本篇文章內容參考如下，如有引用錯誤或須加註問題，歡迎提出!"
+        display:block;
+        color: #947600;
+        padding: 1.25em 1.5em;
+
+.my-toc
+    border: 3px solid #ffdd57;
+    padding-right: 32px;
+    background-color: #fffbeb;
+    border-radius: 5px 5px 5px 5px;
+    width: fit-content;
+    
+
+.blockquote
+    background-color: #eef6fc;
+    border-radius: 4px;
+    padding: 2rem 1.5rem;
+    position: relative;
+    margin-top: 1.6rem;
+    margin-bottom: 1.5rem;
+    &::before 
+        content: "\f10d";
+        top: -20px;
+        margin-right: -16px;
+        right: 100%;
+        font-family: FontAwesome;
+        position: absolute;
+        color: #039be5;
+        font-size: 34px;
+    &::after 
+        content: "\f10e";
+        margin-left: -16px;
+        left: 100%;  
+        top: auto;
+        bottom: -20px;
+        font-family: FontAwesome;
+        position: absolute;
+        color: #039be5;
+        font-size: 34px;
\ No newline at end of file
diff --git a/node_modules/hexo-theme-icarus/include/style/card.styl b/node_modules/hexo-theme-icarus/include/style/card.styl
index db5f46f..1523562 100644
--- a/node_modules/hexo-theme-icarus/include/style/card.styl
+++ b/node_modules/hexo-theme-icarus/include/style/card.styl
@@ -16,3 +16,7 @@
     .media + .media
         border: none
         margin-top: 0
+    
+    &:hover
+        box-shadow: 0 6px 15px rgba(0,0,0,0.15), 0 0 1px rgba(0,0,0,0.1)
+        transform: translateY(-3px);
diff --git a/node_modules/hexo-theme-icarus/include/style/helper.styl b/node_modules/hexo-theme-icarus/include/style/helper.styl
index 84f1141..a572428 100644
--- a/node_modules/hexo-theme-icarus/include/style/helper.styl
+++ b/node_modules/hexo-theme-icarus/include/style/helper.styl
@@ -138,6 +138,14 @@ for n in (0 .. 5)
     &:hover
         color: $primary !important
 
+.category-first-word
+    font-size: 20px;
+    background-color: #96d1f8;
+    padding: 5px;
+    color: white;
+    font-weight: 600;
+    margin-right: 2px;
+
 /* ---------------------------------
  *          Image helpers
  * --------------------------------- */
diff --git a/node_modules/hexo-theme-icarus/layout/common/article.jsx b/node_modules/hexo-theme-icarus/layout/common/article.jsx
index 02e4d4c..396ed32 100644
--- a/node_modules/hexo-theme-icarus/layout/common/article.jsx
+++ b/node_modules/hexo-theme-icarus/layout/common/article.jsx
@@ -1,6 +1,6 @@
 const moment = require('moment');
-const { Component, Fragment } = require('inferno');
-const { toMomentLocale } = require('hexo/lib/plugins/helper/date');
+const {Component, Fragment} = require('inferno');
+const {toMomentLocale} = require('hexo/lib/plugins/helper/date');
 const Share = require('./share');
 const Donates = require('./donates');
 const Comment = require('./comment');
@@ -20,9 +20,9 @@ function getWordCount(content) {
 
 module.exports = class extends Component {
     render() {
-        const { config, helper, page, index } = this.props;
-        const { article, plugins } = config;
-        const { url_for, date, date_xml, __, _p } = helper;
+        const {config, helper, page, index} = this.props;
+        const {article, plugins} = config;
+        const {url_for, date, date_xml, __, _p} = helper;
 
         const indexLaunguage = toMomentLocale(config.language || 'en');
         const language = toMomentLocale(page.lang || page.language || config.language || 'en');
@@ -46,16 +46,19 @@ module.exports = class extends Component {
                     {/* Metadata */}
                     {page.layout !== 'page' ? <div class="article-meta is-size-7 is-uppercase level is-mobile">
                         <div class="level-left">
-                            {/* Creation Date */}
-                            {page.date && <span class="level-item" dangerouslySetInnerHTML={{
-                                __html: _p('article.created_at', `<time dateTime="${date_xml(page.date)}" title="${new Date(page.date).toLocaleString()}">${date(page.date)}</time>`)
-                            }}></span>}
-                            {/* Last Update Date */}
-                            {shouldShowUpdated && <span class="level-item" dangerouslySetInnerHTML={{
-                                __html: _p('article.updated_at', `<time dateTime="${date_xml(page.updated)}" title="${new Date(page.updated).toLocaleString()}">${date(page.updated)}</time>`)
-                            }}></span>}
-                            {/* author */}
-                            {page.author ? <span class="level-item"> {page.author} </span> : null}
+                            {/* Categories first-word*/}
+                            {page.categories && page.categories.length ? <span class="level-item">
+                                {(() => {
+                                    const categories = [];
+                                    page.categories.forEach((category, i) => {
+                                        if (i == 0) {
+                                            categories.push(<span class="category-first-word">{category.name[0]}</span>);
+                                            categories.push(<span> - </span>);
+                                        }
+                                    });
+                                    return categories;
+                                })()}
+                            </span> : null}
                             {/* Categories */}
                             {page.categories && page.categories.length ? <span class="level-item">
                                 {(() => {
@@ -69,6 +72,17 @@ module.exports = class extends Component {
                                     return categories;
                                 })()}
                             </span> : null}
+                            {/* Creation Date */}
+                            {page.date && <span class="level-item" dangerouslySetInnerHTML={{
+                                __html: _p('article.created_at', `<time dateTime="${date_xml(page.date)}" title="${new Date(page.date).toLocaleString()}">${date(page.date)}</time>`)
+                            }}></span>}
+                            {/* Last Update Date */}
+                            {shouldShowUpdated && <span class="level-item" dangerouslySetInnerHTML={{
+                                __html: _p('article.updated_at', `<time dateTime="${date_xml(page.updated)}" title="${new Date(page.updated).toLocaleString()}">${date(page.updated)}</time>`)
+                            }}></span>}
+                            {/* author */}
+                            {page.author ? <span class="level-item"> {page.author} </span> : null}
+
                             {/* Read time */}
                             {article && article.readtime && article.readtime === true ? <span class="level-item">
                                 {(() => {
@@ -87,7 +101,7 @@ module.exports = class extends Component {
                     {page.title !== '' && index ? <p class="title is-3 is-size-4-mobile"><a class="link-muted" href={url_for(page.link || page.path)}>{page.title}</a></p> : null}
                     {page.title !== '' && !index ? <h1 class="title is-3 is-size-4-mobile">{page.title}</h1> : null}
                     {/* Content/Excerpt */}
-                    <div class="content" dangerouslySetInnerHTML={{ __html: index && page.excerpt ? page.excerpt : page.content }}></div>
+                    <div class="content" dangerouslySetInnerHTML={{__html: index && page.excerpt ? page.excerpt : page.content}}></div>
                     {/* Licensing block */}
                     {!index && article && article.licenses && Object.keys(article.licenses)
                         ? <ArticleLicensing.Cacheable page={page} config={config} helper={helper} /> : null}
diff --git a/node_modules/hexo-theme-icarus/layout/common/footer.jsx b/node_modules/hexo-theme-icarus/layout/common/footer.jsx
index fcdecd9..08ce896 100644
--- a/node_modules/hexo-theme-icarus/layout/common/footer.jsx
+++ b/node_modules/hexo-theme-icarus/layout/common/footer.jsx
@@ -1,5 +1,5 @@
-const { Component } = require('inferno');
-const { cacheComponent } = require('hexo-component-inferno/lib/util/cache');
+const {Component} = require('inferno');
+const {cacheComponent} = require('hexo-component-inferno/lib/util/cache');
 
 class Footer extends Component {
     render() {
@@ -35,14 +35,14 @@ class Footer extends Component {
                             {footerLogo}
                         </a>
                         <p class="is-size-7">
-                            <span dangerouslySetInnerHTML={{ __html: `&copy; ${siteYear} ${author || siteTitle}` }}></span>
+                            <span dangerouslySetInnerHTML={{__html: `&copy; ${siteYear} ${author || siteTitle}`}}></span>
                             &nbsp;&nbsp;Powered by <a href="https://hexo.io/" target="_blank" rel="noopener">Hexo</a>&nbsp;&&nbsp;
                             <a href="https://github.com/ppoffice/hexo-theme-icarus" target="_blank" rel="noopener">Icarus</a>
                             {showVisitorCounter ? <br /> : null}
                             {showVisitorCounter ? <span id="busuanzi_container_site_uv"
-                                dangerouslySetInnerHTML={{ __html: visitorCounterTitle }}></span> : null}
+                                dangerouslySetInnerHTML={{__html: visitorCounterTitle}}></span> : null}
                         </p>
-                        {copyright ? <p class="is-size-7" dangerouslySetInnerHTML={{ __html: copyright }}></p> : null}
+                        {copyright ? <p class="is-size-7" dangerouslySetInnerHTML={{__html: copyright}}></p> : null}
                     </div>
                     <div class="level-end">
                         {Object.keys(links).length ? <div class="field has-addons">
@@ -63,9 +63,9 @@ class Footer extends Component {
 }
 
 module.exports = cacheComponent(Footer, 'common.footer', props => {
-    const { config, helper } = props;
-    const { url_for, _p, date } = helper;
-    const { logo, title, author, footer, plugins } = config;
+    const {config, helper} = props;
+    const {url_for, _p, date} = helper;
+    const {logo, title, author, footer, plugins} = config;
 
     const links = {};
     if (footer && footer.links) {
diff --git a/node_modules/hexo-theme-icarus/layout/common/head.jsx b/node_modules/hexo-theme-icarus/layout/common/head.jsx
index 887713d..40f685a 100644
--- a/node_modules/hexo-theme-icarus/layout/common/head.jsx
+++ b/node_modules/hexo-theme-icarus/layout/common/head.jsx
@@ -1,4 +1,4 @@
-const { Component } = require('inferno');
+const {Component} = require('inferno');
 const MetaTags = require('hexo-component-inferno/lib/view/misc/meta');
 const WebApp = require('hexo-component-inferno/lib/view/misc/web_app');
 const OpenGraph = require('hexo-component-inferno/lib/view/misc/open_graph');
@@ -30,8 +30,8 @@ function getPageTitle(page, siteTitle, helper) {
 
 module.exports = class extends Component {
     render() {
-        const { site, config, helper, page } = this.props;
-        const { url_for, cdn, fontcdn, iconcdn, is_post } = helper;
+        const {site, config, helper, page} = this.props;
+        const {url_for, cdn, fontcdn, iconcdn, is_post} = helper;
         const {
             url,
             head = {},
diff --git a/node_modules/hexo-theme-icarus/layout/common/widgets.jsx b/node_modules/hexo-theme-icarus/layout/common/widgets.jsx
index a8f4688..5759085 100644
--- a/node_modules/hexo-theme-icarus/layout/common/widgets.jsx
+++ b/node_modules/hexo-theme-icarus/layout/common/widgets.jsx
@@ -1,5 +1,5 @@
 const logger = require('hexo-log')();
-const { Component } = require('inferno');
+const {Component} = require('inferno');
 const view = require('hexo-component-inferno/lib/core/view');
 const classname = require('hexo-component-inferno/lib/util/classname');
 
@@ -65,7 +65,7 @@ function isColumnSticky(config, position) {
 
 class Widgets extends Component {
     render() {
-        const { site, config, helper, page, position } = this.props;
+        const {site, config, helper, page, position} = this.props;
         const widgets = formatWidgets(config.widgets)[position] || [];
         const columnCount = getColumnCount(config.widgets, config, page);
 
